### DISCUSSION TITLE
How to customize the font style of github bio

### DISCUSSION THREAD

**Discussion by everforyou** [community/community#134826] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I am trying to set italic font in my github profile biography, but it doesn't work well.

Also I was trying to insert line break in bio, but It also didn't work.

I tried using both * and _ , but it doesn't work well. Also, in actually you can confirm in my github account, https://github.com/brotherrabbit91 Is there any method to solve this?

**Comment by ManthanThakor** [comment]
GitHub profiles have limited formatting options for biographies, and Markdown syntax for text styling like italics or line breaks doesn’t work in the GitHub bio section. Here’s a bit more detail:

Italics and Formatting
Markdown formatting (such as italics or bold) is not supported in the GitHub bio section.
You can use plain text only. GitHub’s bio section does not support Markdown or HTML for styling.
Line Breaks
Line breaks are not supported. Text will appear as a continuous block.
Example
Unfortunately, you can’t apply any styles like italics or line breaks directly in your GitHub bio. You’ll need to use plain text to format your biography.

If you need to format text or include additional details, consider:

Using a GitHub README file (in a repository) where Markdown is supported and you can include more detailed formatting.
Including links to external pages or personal sites where you can use full Markdown or HTML.

**Comment by azimtechie** [comment]
you can use basic Markdown styling like bold or italics.

In the bio section, you can use the following Markdown syntax:

Italic: *Your text*

**Reply by everforyou** [reply]
I can't use like that.

**Comment by davevad93** [comment, marked as answer]
Hi @brotherrabbit91 ,

As far as I know GitHub doesn't support styles for the bio, only emojis. You can use Markdown text styles like **bold** or *italic* in the README but `html/css` style properties like `font-family` or `color` aren't supported in GitHub's `md` files, for that you can use [LaTex](https://github.com/orgs/community/discussions/31570#discussioncomment-3571340).

**Reply by everforyou** [reply]
Thanks, but i don't mean MD file.

**Reply by davevad93** [reply]
You're welcome @brotherrabbit91 , I guess you meant the profile bio below the user picture; it only supports emojis but not font styles, colors or text formatting.

**Reply by everforyou** [reply]
Thanks.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @brotherrabbit91 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Profile category. I've gone ahead and moved it for you. Good luck!
